src/sentry/static/sentry/app/components/events/interfaces/keyValueList/keyValueListV2.tsx

```diff
@@ -17,7 +17,7 @@ type Props = {
 
 export type KeyValueListData = {
   key: string;
-  subject: React.ReactNode;
+  subject: React.ReactNode | string;
```

Comment:
I think string is already part of ReactNode.

Reply:
Yes, string is part of React.ReactNode.